### PR TITLE
gl-stitch: set up gl copy shader

### PR DESCRIPTION
**Pull request by liuyinhangx**

### CODE REVIEW

File: modules/gles/Makefile.am [hunk 1]
```diff
@@ -12,6 +12,7 @@ XCAM_GLES_LIBS =                     \
     $(NULL)
 
 xcam_gles_sources =                  \
+    gl_error.cpp                     \
```
Comment (windyuan) [inline modules/gles/Makefile.am]: no need to separate new file gl_error.h / .cpp.
move all gl related utilities to gles_std.h or gles_common.h

File: modules/gles/Makefile.am [hunk 2]
```diff
@@ -19,6 +19,7 @@ xcam_gles_sources =                  \
     gl_video_buffer.cpp              \
     gl_compute_program.cpp           \
     gl_command.cpp                   \
+    gl_utils.cpp                     \
```
Comment (windyuan) [inline modules/gles/Makefile.am]: merge gl_utils and gl_error.h into one.
e.g. use gl_common.h or gl_utils.h as new file name.

**Comment by liuyinhangx** [comment]
update patch, please help to review, thanks.

**Comment by windyuan** [comment]
merged. thanks.

**Pull request merged**